### ISSUE TITLE
Block of Steel vanished after trying to take it out with a chute or a hopper.

### ISSUE THREAD

**Issue by Yamadaisme** [DrMango14/Create-The_Factory_Must_Grow#55]
I followed the recipe using casting basin and a casting spout to pour in the molten steel with a hopper below to get the item out. After a the block hardened the block just disappeared from the basin without outputing any item out of the hopper.

**Comment by ExpoTr** [comment]
same bug

**Comment by ParzivalPotaru** [comment]
same issue here

**Comment by russinet** [comment]
yup, people on my server are losing their steel! The thing is that once it starts happening, it's happening for **everyone** on the server, all the ingots just disappear. The only way to mitigate this is to restart the server, but that's extremely inconvenient.

**Comment by Eliaz-LR** [comment]
Same here, restarting the server solves it for like a minute and then it becomes buggy again

**Comment by DrMango14** [comment]
fixed
